bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAirbaseUnitHandler.java

```diff
@@ -128,14 +128,25 @@ protected void changeSetPoint(double newTemperature) throws DaikinCommunicationE
 
     @Override
     protected void changeMode(String mode) throws DaikinCommunicationException {
+        AirbaseMode newMode = AirbaseMode.valueOf(mode);
+        if ((newMode == AirbaseMode.AUTO && !airbaseModelInfo.features.contains(AirbaseFeature.AUTO)) ||
+            (newMode == AirbaseMode.DRY  && !airbaseModelInfo.features.contains(AirbaseFeature.DRY))) {
+            throw new IllegalArgumentException("The " + mode + " mode is not supported by your controller");
```

Comment:
Who will catch that exception?

Reply:
Nobody. It will show up in the log. Should I just issue a WARN/ERROR log instead? If so, should it be a WARN or an ERROR?

Reply:
The question is: Is this happening due to a mis-configuration by the user and prevents proper binding functionality (then `WARN`), is it just a "wrong" command (then `INFO`) or can it be silently ignored and no one cares (then `DEBUG`).

Reply:
I am torn between WARN vs INFO. It is essentially a "wrong" command, but I feel that it needs to be highlighted to the user so that they will correct it, so I am opting for WARN.